content/en/synthetics/guide/synthetic-test-monitors.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Understanding Synthetic Tests and Alerting Monitors
+kind: guide
+description: Learn about Synthetic monitors created with your Synthetic tests. 
+further_reading:
+- link: "/monitors/manage/"
+  tag: "Documentation"
+  text: "Manage Monitors"
+- link: "/synthetics/metrics/"
+  tag: "Documentation"
+  text: "Synthetic Monitoring Metrics"
+---
+
+## Overview
+
+When you create a Synthetic test, Datadog automatically creates an associated monitor. The monitor's alerting conditions in your test configuration impact how you are notified when a Synthetic test moves into an alerting state.
+
+{{< img src="synthetics/guide/synthetics_test_monitors/synthetic_test_monitor.png" alt="Synthetic test monitor" style="width:100%;">}}
+
+This guide describes how Synthetic test monitors behave, how to use a monitor's conditional variables in a Synthetic test configuration, and how to set downtimes and mute monitors.
+
+## Create a Synthetic test monitor
+
+<div class="alert alert-info">Alerting monitors created by Synthetic tests are not a <a href="/monitors/create/">monitor type</a>. You cannot create or import a Synthetic test monitor.</div>
+
+Following the test creation workflow, create a monitor in the **Configure the monitor for this test** section to send notifications when a Synthetic test is failing. The Synthetic test monitor integrates with notification channels such as email, Slack, Pagerduty, and Microsoft Teams.
+
+{{< img src="synthetics/guide/synthetics_test_monitors/configure_the_monitor_for_this_test.png" alt="Creating a monitor in your Synthetic test" style="width:90%;">}}
+
+Customize the monitor name to identify it on the [**Manage Monitors**][1] page. You can also use [conditional variables][2] to characterize the notification message based on test state.
+
+Synthetic test monitors do not support `{{#is_warning}}`, `{{^is_warning}}`, `{{#is_warning_recovery}}`, and `{{^is_warning_recovery}}` conditional variables. Datadog recommends enabling [renotification][3] on your monitors.
+
+## Notify multiple teams with monitor renotifications
+
+Depending on your incident management strategy, you may want to involve multiple teams when a Synthetic test alerts. You can use the available conditional variables to set which team are notified in your notification message. 
+
+To enable renotifications, click the toggle left of `If this monitor stays in alert status renotify every` and select a time option from the dropdown menu.
+
+## Integrate your Synthetic test monitor with Statuspage
+
+If you use [Atlassian Statuspage][4] for visibility into your applications' and services' uptime, you can update the status of your systems with  Synthetic test monitor notifications.
+
+1. See the [Statuspage documentation][5] to generate a component-specific email address.
+2. Add the generated email address into your test's notification message.
+3. Customize the monitor name to return `UP` or `DOWN` depending on the test state.
```

Comment:
You might want to mention conditional variables here to make it clear to users how they can get this working without having them look into the screenshot.